.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
```

Comment:
What does this actually do, though? Do we need to define dependencies somewhere? Or what even are dependencies in this case?

Reply:
Ah I probably should've added an explanation, it checks any external actions used in the workflows, which if you look in the workflow `.yml` file look like: `- uses: actions/checkout@v2` and exist as Github repos, for example https://github.com/actions/checkout.

It checks if there are newer versions available. So in that example if there was a `v3` release of the checkout action available Dependabot would create a PR here automatically to bump the version inside the workflow file and keep it up to date.